iroha_2_whitepaper.md

```diff
@@ -88,11 +94,18 @@ For the decentralized membership service, the following transaction types are ap
 * AddCurveLiquidity(asset, asset)
 * ChainedTrade(inputAsset, outputAsset) # Does paythingfinding to get output; oraclized inputs/outputs are special
 
+* addLimitBid
+* addLimitAsk
+* removeLimitBid
+* removeLimitAsk
```

Comment:
We also already have a lot of ISIs of type actionPARAMETERentity, maybe we can already start generalizing some instructions, like `Add(WhatToAdd, WhereToAdd)` and provide an ability to compose them:
```
Transfer = Sub(AssetAmount, Account1) -> Add(AssetAmount, Account2)
```
?